packages/create-stylable-app/template/ts-react-webpack/template.js

```diff
@@ -18,8 +18,8 @@ module.exports = {
         'serve',
         'ts-loader',
         'typescript',
-        'webpack@4',
-        'webpack-cli@3',
+        'webpack',
```

Comment:
remember to change `html-webpack-plugin` to `@next`

Reply:
it's in the task list